README.md

```diff
@@ -1 +1,28 @@
-# trailmap
\ No newline at end of file
+# trailmap
+
+This tool takes a Cyclus input file and conducts acquisition pathway analysis on the facilities specified.
+
+## Dependencies
+
+* Python 3
+* [Cyclus](https://github.com/cyclus/cyclus)
+* [Cycamore](https://github.com/cyclus/cycamore)
+* [NetworkX](https://networkx.github.io/)
+* [PyGraphviz](https://pygraphviz.github.io/)
+* [pydot](https://github.com/pydot/pydot)
+
+Follow installation instructions from [Cyclus](https://github.com/cyclus/cyclus) and [Cycamore](https://github.com/cyclus/cycamore).
+
+Install required python packags
+
+`python -m install --user networkx pygraphviz pydot`
```

Comment:
This line doesn't quite look right:
```suggestion
`python -m pip install --user networkx pygraphviz pydot`
```

Reply:
also can't these just run pip diretly:
`pip install --user networkx pygraphviz pydot`

Reply:
I was told that using `python -m pip`  is good practice because it isn't ambiguous and won't accidentally use the last copy of pip you installed, which might not be the right pip for the version of python. I always `python -m pip` partially because I've had that problem in the past when I have multiple versions of python installed on my machine

https://snarky.ca/why-you-should-use-python-m-pip/